.github/actions/expo-caches/action.yml

```diff
@@ -225,3 +228,15 @@ runs:
         # Reset reactNativeArchitectures to build all architectures
         ORG_GRADLE_PROJECT_reactNativeArchitectures:
       working-directory: android
+
+    - name: 🛠 Setup "REACT_NATIVE_DOWNLOADS_DIR" environment variable
+      if: inputs.react-native-gradle-downloads == 'true'
+      shell: bash
+      run: echo "REACT_NATIVE_DOWNLOADS_DIR=$HOME/.gradle/react-native-downloads" >> $GITHUB_ENV
+    - name: ♻️ Restore Gradle downloads cache for React Native libraries
+      if: inputs.react-native-gradle-downloads == 'true'
+      uses: actions/cache@v2
+      with:
+        path: ${{ env.REACT_NATIVE_DOWNLOADS_DIR }}
+        key: ${{ runner.os }}-gradle-downloads-${{ hashFiles('yarn.lock') }}
```

Comment:
Hmm, maybe `runner.os` is not necessary? 🤔

Reply:
sounds good! resolved in 85a8f00